src/components/EditorMarkdownIt.vue

```diff
@@ -143,6 +145,13 @@ export default {
 				}
 			}
 		},
+
+		setSingleCodeRule() {
+			this.md.renderer.rules.code_inline = function(tokens, idx, options, env, self) {
+				const token = tokens[idx]
+				return '<p class="singleLineCode"><code>' + token.content + '</code></p>'
```

Comment:
I don't know much about the development process and used transpilers, but
1. is it possible to use javascript template strings for a more readable code?
2. do we need to check whether `token.content` is available? something like `token?.content` or similar?

Reply:
1. Until now, i did not know that javascript HAD template strings, thats a nice find :D
2. I dont think so. When i understood the MarkdownIT library correctly, token can never be undefined. content may be, but that would seem like a bug, but i will improve null-safety!